pkg/pod/status_test.go

```diff
@@ -1248,6 +1250,51 @@ func TestMakeTaskRunStatusAlpha(t *testing.T) {
 				CompletionTime: &metav1.Time{Time: time.Now()},
 			},
 		},
+	}, {
+		desc: "test result types mismatch",
```

Comment:
are there any test cases for successful matches?

Reply:
thanks! Just added